br/pkg/version/version.go

```diff
@@ -32,7 +33,11 @@ var (
 
 // NextMajorVersion returns the next major version.
 func NextMajorVersion() semver.Version {
-	nextMajorVersion := semver.New(removeVAndHash(build.ReleaseVersion))
+	nextMajorVersion, err := semver.NewVersion(removeVAndHash(build.ReleaseVersion))
+	if err != nil {
+		// build.ReleaseVersion is unknown, assuming infinitely-new nightly version.
+		return semver.Version{Major: math.MaxInt64, PreRelease: "nightly"}
```

Comment:
```suggestion
		return semver.Version{PreRelease: "nightly"}
```
I think we should make next version to 0 so it will fail all the version check.

Reply:
BTW, shall we output a `WARN` log here